ext/atomic/index.md

```diff
@@ -0,0 +1,510 @@
+  the local identity extension.
+
+- `href`: a string that contains a URI-reference [[RFC3986 Section
+  4.1](https://tools.ietf.org/html/rfc3986#section-4.1)] that identifies the
+  target of the operation.
+
+An operation object **MAY** also contain any of the following members:
+
+- `data`: the operation's "primary data".
+
+- `meta`: a [meta object](#document-meta) that contains non-standard
+  meta-information about the operation.
+
+Different members are required for processing different types of operations, as
+described below.
+
+### <a href="#result-objects" id="result-objects" class="headerlink"></a> Result Objects
+
+An operation result object **MAY** contain any of the following members:
+
+- `data`: the "primary data" resulting from the operation.
+
+- `meta`: a [meta object](#document-meta) that contains non-standard
+  meta-information about the result.
+
+An empty result object (`{}`) is acceptable for operations that are not required
+to return `data`.
+
+## <a href="#processing" id="processing" class="headerlink"></a> Processing
+
+All HTTP requests sent with this extension **MUST** be issued with `POST`.
+
+A server **MUST** perform operations in the order they appear in the
+`atomic:operations` array.
+
+A server **MUST** perform all operations atomically, so that a failure
+to perform any operation **MUST** invalidate any effects of preceding
+operations.
+
+A server **MUST** respond to a successful operations request with `200 OK` if a
+response document is returned. An array of [result objects](#result-objects)
+**MUST** be returned in a top-level `atomic:results` member. The results array
+**MUST** be the same length as the requested operations array and each result
+**MUST** correspond positionally to its associated operation.
+
+A server **MAY** respond to successful requests that include operations with
+`204 No Content` and no response document if no operations are required to
+return `data`.
+
+### <a href="#processing-errors" id="processing-errors" class="headerlink"></a> Processing Errors
+
+If any operation in a request fails, the server **MUST** respond as described
+[in the base specification](/format/#errors-processing). An array of one or
+more [error objects](/format/#error-objects) **SHOULD** be returned, each with
+a `source` member that contains a `pointer` to the source of the problem in the
+request document.
+
+### <a href="#specific-operations" id="specific-operations" class="headerlink"></a> Processing Specific Operations
+
+The following sections describe how to process specific operations.
+
+#### Creating Resources
+
+An operation that creates a resource **MAY** target a resource collection
+through the operation's `ref` member. The operation **MUST** include an `op`
+code of `"add"` as well as a resource object as `data`. The resource object
+**MUST** contain at least a `type` member.
+
+For example:
+
+```json
+POST /bulk HTTP/1.1
```

Comment:
Maybe change `/bulk` to `/operations` to reflect intended usage.

Reply:
```suggestion
POST /operations HTTP/1.1
```

I think that makes sense.

Reply:
I agree that `/operations` makes for a better example 👍